.github/workflows/ci.yml

```diff
@@ -3,7 +3,13 @@ name: CI # Run CI Checks
 on:
   pull_request:
     branches:
+<<<<<<< HEAD
```

Comment:
^^ Guessing this was leftover from some merge conflict

Reply:
Oh man, good catch! Let me comb through and make sure there aren't any more of these. Weird that it made it through the pre-commit hooks.